haiku-apps/sync_modular/sync_modular-2.2.1b.recipe

```diff
@@ -0,0 +1,79 @@
+SUMMARY="Native, modular software synthesizer"
+DESCRIPTION="It gives you, though limited as any synth, but quite versatile possibilities for creating \
+and using new sounds.
+
+Registration:
+User name : Free registration
+Serial number : eCDiPa-kbbmPZ-zGDJuyU
```

Comment:
I'm not sure... Development of SM was stopped a long time ago. It's quite improbable that its status of being Freeware changes. It's more likely that the website one day just disappears. Finding the old binary somewhere to point SOURCE_URI to probably won't be the problem. Obtaining the freeware registration could be...